Jenkinsfile

```diff
@@ -463,13 +463,17 @@ def convergeCoverage() {
   dir("${BASE_DIR}"){
     def matrixDump = pythonTasksGen.dumpMatrix("-")
     for(vector in matrixDump) {
-      unstash("coverage-${vector}")
+      catchError {
```

Comment:
Why don't they exist?

Reply:
https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#catcherror-catch-error-and-set-build-result-to-failure

Reply:
Yeah, I know :) my point is about why we are using a key that could not exist: `coverage-${vector}`. At some point we should have stashed that key,

Reply:
if for some reason(build failed on the test stage) there are not coverage files, the stash would not store anything so the unstash files will fail.